Source/FakeItEasy/Core/DefaultFakeWrapperConfigurer.cs

```diff
@@ -6,21 +6,27 @@ namespace FakeItEasy.Core
     /// Handles configuring of fake objects to delegate all their calls to a wrapped instance.
     /// </summary>
     internal class DefaultFakeWrapperConfigurer
-        : IFakeWrapperConfigurer

```

Comment:
Good idea.
I noticed the tests `DefaultFakeWrapperConfigurator`, and "Configurator" is more prevalent in the solution than Configurer (this is the only type name containing "configurer"). So `FakeWrapperConfigurator`?
I could give `DynamicContainerTests.availableConfigurers` the same treatment while I'm at it.